tools/devpack.ps1

```diff
@@ -38,42 +47,44 @@ if (!(Test-Path $localPack))
   New-Item -Path $localPack -ItemType directory | Out-Null
 }
 Write-Host
-Write-Host "---Updating project with local SDK pack---"
+Write-Host "---Updating projects with local SDK pack---"
 Write-Host "Packing Core .NET Worker projects to $localPack"
-& "dotnet" "pack" $sdkProject "-p:PackageOutputPath=$localPack" "-nologo" "-p:BuildNumber=$buildNumber" $AdditionalPackArgs
+& "dotnet" "pack" $sdkProject "-p:PackageOutputPath=$localPack" "-nologo" "-p:Version=2.0.1" "-p:VersionSuffix=$buildNumber" $AdditionalPackArgs
```

Comment:
is this version meant to always be hardcoded?

Reply:
No, I don't intend for this to be permenant. It's just a funky state where aspnetcore bits are all on 2.0.0. Then the test script builds 2.0.0-local20250125 which is considered a "downgrade" from 2.0.0 so the build and pipeline fail. I should be releasing 2.1.0 after this PR and my plan is to switch back to what we had before `"-p:BuildNumber=$buildNumber"` which should continue to work going forward. I just didn't feel like spending time on version # parsing was worth the time for something like this.